src/app/public/modules/phone-field/phone-field-input.directive.ts

```diff
@@ -262,9 +262,13 @@ export class SkyPhoneFieldInputDirective implements OnInit, OnDestroy, AfterView
 
     if (this.phoneFieldComponent.selectedCountry && !this.validateNumber(value)) {
 
-      // Mark the invalid control as touched so that the input's invalid CSS styles appear.
-      // (This is only required when the invalid value is set by the FormControl constructor.)
-      control.markAsTouched();
+      if (document.activeElement !== this.elRef.nativeElement) {
+        // Mark the invalid control as touched so that the input's invalid CSS styles appear.
+        // (This is only required when the invalid value is set by the FormControl constructor.)
+        // We don't do this if the input is the active element so that we don't show validation
+        // errors unless it is invalid on initialization or the input has been blurred.
+        control.markAsTouched();
```

Comment:
Why not just remove this line altogether? The "touched" state gets set by the `public onInputBlur(): void` handler, above.

Reply:
The line is there so that we mark the field as "touched" when the form is initialized to an invalid value. Without that, the control will not mark itself as "dirty" and have the invalid stylings in this case.

Reply:
We do this in Datepicker as well. We don't have this extra if-statement in Datepicker because we don't validate that control while typing. However, Todd wanted phone field to validate while you type (with a time of half a second on line 285)

Reply:
I see. I'd prefer to find a solution that doesn't require interaction with the `document`, if possible. Maybe a boolean check after the input is initially blurred?

Reply:
I've changed this to look for the `textChanges` behavior subject that is only added once a user interacts with the input. Let me know if you want to chat about this.